swarm/pss/pss.go

```diff
@@ -748,6 +751,23 @@ func (self *Pss) forward(msg *PssMsg) {
 // SECTION: Caching
 /////////////////////////////////////////////////////////////////////
 
+func (self *Pss) getFwdCacheSize() int {
+	self.fwdCacheMu.Lock()
+	defer self.fwdCacheMu.Unlock()
+	return len(self.fwdCache)
+}
+
```

Comment:
Please don't add methods in the implementation code just for test. You can access private members directly.